drivers/bridge/bridge.go

```diff
@@ -122,7 +125,11 @@ func Init(dc driverapi.DriverCallback) error {
 	c := driverapi.Capability{
 		Scope: driverapi.LocalScope,
 	}
-	return dc.RegisterDriver(networkType, newDriver(), c)
+	err2 := dc.RegisterDriver(networkType, newDriver(), c)
+	if err2 != nil && err1 != nil {
```

Comment:
Thanks for checking for the bridge module before probing for it.
But, I still not clear on why a failure to probe on `nf_nat` is combined with RegisterDriver instead of warning the user right when the probe fails ?